pkg/limayaml/limayaml.go

```diff
@@ -9,6 +9,7 @@ import (
 type LimaYAML struct {
 	Arch              *Arch             `yaml:"arch,omitempty" json:"arch,omitempty"`
 	Images            []File            `yaml:"images" json:"images"` // REQUIRED
+	CPUType           *string           `yaml:"cpuType,omitempty" json:"cpuType,omitempty"`
```

Comment:
I already merged this PR, but on second thought we could define this as `[]struct{Type string, Arch Arch}` so that a single YAML can be used on multiple archs?

Reply:
Changed to `map[Arch]string` in https://github.com/lima-vm/lima/pull/656 (thanks @jandubois )